examples/campaign_management/add_campaign_labels.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This code example adds a campaign label to a list of campaigns.
+
+This example assumes that a label has already been prepared.
+"""
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, label_id, campaign_ids):
+    """This code example adds a campaign label to a list of campaigns.
+
+    Args:
+        client: An initialized GoogleAdsClient instance.
+        customer_id: A client customer ID str.
+        label_id: A label ID to use when searching for campaigns.
```

Comment:
This description seems wrong, in PHP we use "the ID of the label to attach to campaigns"

Reply:
Fixed.  I had brought the original over from other examples to match the style and overlooked the description.